### PR TITLE
[ZH] Update Ranking Criteria for each game mode

### PR DESCRIPTION
**Pull request by huoyaoyuan**
Including difficulty naming.

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!mania/zh.md [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
 - **Musical phrase（乐句）**： 音乐中表示一个独立单元的计量单位，一般是四个小节长。
 - **Short-term slider velocity （短距滑条变速）**： 一个持续 1/2 拍以下的滑条变速。
 
 ### 梗
 
 - **Anchor**： 一种在一列中放置比其他列明显多很多物件的梗，经常出现于不同种类的连打中。
+- **Bracket**： 在多组相邻的列间同时的交互。
 - **Chord（多押）**： 同时按下两个或者更多的物件。
 - **Chordstream（多押连打） （Jumpstream, Handstream, Quadstream）（双押连打，三押连打，四押连打）**： 一种混合各种种类的多押连打。
 - **Grace**： 快速接连按下 2 个或两个以上的物件，通常使用大于等于 1/6 的节拍。
```
Comment (mcendu) [inline wiki/Ranking_Criteria/osu!mania/zh.md]: Jack = 纵连

File: wiki/Ranking_Criteria/osu!mania/zh.md [hunk 2]
```diff
@@ -45,29 +44,26 @@ outdated: true
 - **Shield**： 在释放一个长物件后立即在相同列出现一个物件。
```
Comment (mcendu) [inline wiki/Ranking_Criteria/osu!mania/zh.md]: Jumptrill = 双押交互

File: wiki/Ranking_Criteria/osu!/zh.md [hunk 3]
```diff
@@ -1,216 +1,202 @@
...
-- **Hit object/Note density：** 用节拍数表示的物件之间的平均时间跨度。（比如 1/1 表示一拍，1/2 表示半拍）。
+- **Slider anchors（滑条锚点）：** 用于固定滑条形状的工具，灰色锚点用于让滑条产生一个弧度，而红色锚点用于结束这里的弧度。也被称作「Nodes」。
+- **Slider velocity（滑条速度）：** 滑条的速度决定了在给定的节拍内滑条会经过多长的距离。谱面基础的滑条速度可以在 Timing 界面设置，另外的滑条速度变更可以通过继承时间点（绿线）来完成。
+- **Short reversing slider（短折返滑条）：** 1 次或者多折返的短滑条（通常是 1/4 或者 1/8），使玩家可以将光标停留在一个地方来完成折返。
+- **Extended slider（延长滑条）：** 尾巴不在音乐特定位置上的滑条，但是它表达延长音，通常的用法是 3/4 拍滑条。
+- **Slider tick rate（滑条打点率）：** 音乐中每一拍出现的滑条点的数量。
+- **Spinner recovery time（转盘恢复时间）：** 转盘结束到下一个物件开始之间的时间。
+- **Time-distance equality（时间 - 间距等比例）：** 物件之间的间距与物件相隔的时间成固定比例。也叫固定间距。
+- **Hit object/Note density（物件密度）：** 物件之间的平均时间跨度，用节拍数表示。（比如 1/1 表示一拍，1/2 表示半拍）。
 - **Keysounding：** 使用与音乐很相近的音，或者直接用音乐中的音来作为谱面音效。 这种音效的做法通常给玩家的打击反馈感很少。
```
Comment (mcendu) [inline wiki/Ranking_Criteria/osu!/zh.md]: 即“key音”

File: wiki/Ranking_Criteria/osu!/zh.md [hunk 4]
```diff
@@ -1,216 +1,202 @@
...
 #### 指南
 
--  AR 应该不超过 5。
--  OD/HP 应该在 1-3 之间。
+- **应使用时间 - 间距等比例设定。** 如果间距明显与其他节奏使用的间距不同的话可以接受适当的变化。当物件之间相隔 2 拍以上，且在游戏区域内的间距相当远，则间距不需要这样精确。
+- **当使用 distance snap 时，尽量将倍率保持在 0.8x 到 1.2x 之间。** 过高或过低的数值会使视觉上滑条速度和单点间距间产生反直觉。高 DS 可以用来避免物件接触，但是如果你经常需要增大 DS，那么你应该同时增加滑条速度。
+- **物件密度应主要由 1/1 和 2/1 节奏或者更慢的节奏组成。**
+- **避免使用比 1/2 滑条更短的滑条。** 这些滑条对于新手玩家来说太过快速，不易理解。
+- **避免圆圈、滑条头以及滑条尾互相接触。** 这些会困扰新手玩家。对于 1/1 节奏下由 DS 产生的物件接触，应该使用更大的间距或者滑条速度来避免。
+- **避免使用多次折返的 1/1 滑条。** 额外的折返可见的时间很短，所以新手玩家无法读清。
```
Comment (mcendu) [inline wiki/Ranking_Criteria/osu!/zh.md]:
```suggestion
- **避免使用多次折返的 1/1 滑条。** 额外的折返可见的时间极短，这会导致新手无法读清。
```

**Comment by cl8n** [comment]
small update to taiko #3374

**Comment by huoyaoyuan** [comment]
是的，因为这几篇之前是反过来的，所以我没有注意到。
Yes. These articles all used a reversed format so I didn't mention that.

**Pull request merged**